.travis.yml

```diff
@@ -1,32 +1,22 @@
-language: node_js
-node_js:
-  - "node"
+language: python
+python:
+  - "3.7"
 cache: false
-addons:
-  apt:
-    packages:
-    - calibre
-    - calibre-bin
 dist: xenial
-services:
-  - xvfb
-before_install:
-  - npm install gitbook-cli -g
+
 install:
-  - gitbook install
-  - npm install svgexport -g
-# Using xvfb to Run Tests That Require a GUI
-# https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI
-before_script:
-  - "export DISPLAY=:99.0"
-  - sleep 3 # give xvfb some time to start
+  # FIXME: Inline math is broken in upstream recommonmark
+  - pip install git+https://github.com/chrisburr/recommonmark.git@patch-1
+  - pip install starterkit-ci
```

Comment:
What is in starterkit-ci? Theme and some runners? Will it be here too (since it needs to also be maintained)?

Reply:
Yes and it's still present in some of the earlier commits. I've just made the repository for it and pushed it here: https://github.com/lhcb/starterkit-ci